client/my-sites/plans-features-main/index.tsx

```diff
@@ -817,9 +832,21 @@ const PlansFeaturesMain = ( {
 				{ isPlansGridReady && (
 					<>
 						{ ! hidePlanSelector && (
-							<div className="plans-features-main__plan-type-selector">
-								<PlanTypeSelector { ...planTypeSelectorProps } />
-							</div>
+							<StickyContainer
+								stickyClass="is-sticky-plan-type-selector"
```

Comment:
I don't quite follow why you need this and not something you can't add below in `className`? 🤔

Reply:
The stickyClass is only applied when the the child is actually stickied to the screen